python/ml4ir/base/data/tfrecord_writer.py

```diff
@@ -1,8 +1,43 @@
+"""
```

Comment:
This is an initiative to update the file. It uses a bit my taste, but I made sure to not break the `write_from_files` and `write_from_file` function APIs. But I changed the API of the command line tool, adding support for both tf.example and tf.sequence etc.

Reply:
Wonderful. I missed updating this as part of my huge refactor work.